### ISSUE TITLE
Make it possible to exclude archaic definitions

### ISSUE THREAD

**Issue by r00ster91** [FooSoft/yomichan#445]
It is quite rare that archaic definitions are helpful, so how about having an option to exclude them from the results?

**Comment by siikamiika** [comment]
I think the simplest solution would be to create a separate dictionary that doesn't contain the archaic definitions. It would require a custom build of Yomichan Import or some other way of editing existing dictionaries. The source for the JMdict exporter is here: https://github.com/FooSoft/yomichan-import/blob/master/edict.go

If you think other dynamic filters would be useful, it's also possible to add this feature to Yomichan, but that's more complex to implement and a little bit slower.

**Comment by r00ster91** [comment]
Yeah, that sounds like the simplest solution. But of course this could also be extended to having general filters with more options than just archaic definitions.
